pkg/authenticator/authenticator.go

```diff
@@ -254,6 +263,25 @@ func (auth *Authenticator) ParseAuthenticationResponse(response []byte) error {
 	return nil
 }
 
+func waitForCertificate(certificatePath string, timeout time.Duration) error {
+	deadline := time.Now().Add(timeout)
```

Comment:
We already use the [`backoff` library]("github.com/cenkalti/backoff") in the [main loop](https://github.com/cyberark/conjur-authn-k8s-client/blob/master/cmd/authenticator/main.go#L35-L43). You might as well use the same lib for the logic.

Reply:
I know it's a little edge case, but since we only care about calculating elapsed time, can we use UTC so that it will not be affected by daylight saving time? I want to avoid unexpected behavior that could occur for 2 hours a year.